_posts/nik-slien01

```diff
@@ -0,0 +1 @@
+welcome to github learn
```

Comment:
---
layout: slide
title: "Welcome to our second slide!"
---
Your text
Use the left arrow to go back!